data_model/src/query/mod.rs

```diff
@@ -179,12 +179,11 @@ pub mod model {
         Debug, Clone, PartialEq, Eq, Getters, Decode, Encode, Deserialize, Serialize, IntoSchema,
     )]
     #[getset(get = "pub")]
-    #[ffi_type]
     pub struct TransactionQueryOutput {
         /// The hash of the block to which `tx` belongs to
         pub block_hash: HashOf<SignedBlock>,
-        /// Transaction
-        pub transaction: TransactionValue,
+        /// Transaction. Boxed to not to have infinite size.
```

Comment:
Why would it have infinite size? Both fields of `TransactionValue` are sized, no?

Reply:
Answered [here](https://github.com/hyperledger/iroha/pull/4089#discussion_r1410278751)

Reply:
```suggestion
        /// Transaction
```

we don't need these vague comments. It will be clear to anyone who tries to remove the `Box` why it's needed